skeleton.py

```diff
@@ -175,6 +184,9 @@ def setEntity(self, entity: db.Entity):
 		self.accessedValues = {}
 		self.renderAccessedValues = {}
 
+	def getValues(self) -> Dict[str, Any]:
```

Comment:
This should probably not be included. I'd had this discussion recently. getValues() was a hack to retrieve a stable dictionary of key<->values pairs for later usage (as the contents inside the skel could magically change when iterating over a skellist). This is guaranteed not to happen in Py3, so you should be able to use the skel itself in most cases - which is much faster as this expression forces the deserializion of all bones - even if they would not have been accessed otherwise.

Reply:
Hmm... I think we should discuss this in the next ViUR round.
I'am using this method often for debugging. If your print/log the _Skeleton_ instance you get only a dictionary of the bone names and bone instances - but with _getValues_ you always get the unserialized values and can straightforward see all values.